src/main/scala/com/evolutiongaming/conhub/SequentialMapHelper.scala

```diff
@@ -25,7 +24,7 @@ object SequentialMapHelper {
       value: V,
       onUpdated: (K, Set[V], Set[V]) => Unit = (_, _, _) => ()): Future[Unit] = {
 
-      implicit val ec = CurrentThreadExecutionContext
+      implicit val ec: ExecutionContext = ExecutionContext.parasitic
```

Comment:
Why do we need to change `ExecutionContext`? It this something we need for Scala 3?

Reply:
Nvm, I checked the code in `execution-tools`, `CurrentThreadExecutionContext` simply delegates to `parasitic`, make sense